File: Problem Solving/Stepping Stones Game.c
#include <assert.h>
#include <limits.h>
#include <math.h>
#include <stdbool.h>
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main()
{
    unsigned long long t,n,i,j,k=0,p;
    scanf("%lld", &t);
    for(i=0;i<t;i++)
    {
        p=0;
        scanf("%lld", &n);
        for(j=1;j<n;j++)
        {
            k=k+j;
            if(n==k)
            {
              p=1;
              break;
            }
        }
        if(p==1)
        {
            printf("Go On Bob %lld\n", j);
        }
        else
        {
            printf("Better Luck Next Time\n");
        }
    }
}

File: Python/Stepping Stones Game.py
n=int(input())
jump=1
u=0
arr=[(x*(x+1))/2 for x in range(1, 10000)]
for i in range(n):
    num=int(input())
    for j in range(len(arr)):
        if num==arr[j]:
            u=1
            break
            
if u==1:
    print("Go On Bob " + str(j+1))

else:
    print("Better Luck Next Time")